contents/docs/user-guides/support.md

```diff
@@ -0,0 +1,143 @@
+
+From our side we will select the right team for you - typically this would be:
+
+* Customer Success contact (whoever you spoke with ahead of becomign a paying customer)
+* Support hero engineer (we have an on call engineer between 2am PT / 9am GMT and 5pm PT / 12am GMT)
+* Platform team engineers (help with deeper infrastructure or ingestion questions)
+* Product manager (helps us to understand feature requests, or can clarify current feature set)
+
+[These are the kinds of people](/handbook/company/team) you'd be working with.
+
+Notice - we do *not* have non technical support people who triage issues. We directly connect you to people who are building, designing or who sold the deal. This has scaled to support thousands of companies very effectively - we always aim to productize solutions to problems.
+
+From your side, it's important you bring your A team. We recommend you add:
+
+* A stakeholder from engineering
+* A stakeholder from data engineering / data analysis if relevant
+* A stakeholder from product management
+* 2-3 power users
+* A stakeholder from customer success if relevant
+* A stakeholder from sales if relevant
+
+It's best not to let your team get too much bigger as it can create a lot of noise. We don't support for example you adding 125 end users of PostHog to this group - you need to triage your own issues with larger volumes.
+
+## Dedicated email
+
+For Enterprise customers that can't use Slack easily, we can offer priority email support instead. 
+
+## Training sessions
+
+We run a set number of group training sessions with your end users. We will walk them through:
+
+* Understanding product analytics performance
+* Diagnosing why your analytics are performing well or badly
+* Rolling out changes safely
+* Optional: getting event data into PostHog / out of PostHog
+
+These are useful at first for the initial adoption of the product. As you organically add more people to the platform, we can make sure everyone understands how to drive the most value.
+
+## Deployment developer pairing
+
+We generally do not have access to the infrastructure of our customers. Deployment developer pairing means we can have a developer join a call with one of your developers (ideally with a screenshare, although we can do audio only), to help you quickly get up and running. This is in addition to our extensive [documentation](/docs).
+
+The objective is that you spend as little time as possible worrying about deploying PostHog.
+
+## Configuration assistance
+
+We help you define meaningful dashboards for your company and regularly check in that the configuration of PostHog is helping you improve your product.
+
+We ask to be a user in your PostHog team so we can suggest ways to get more value out of the product. Typically this means ensuring teams build out:
```

Comment:
This sounds more structured than reality - do we have a calendar or a checklist for clients to make sure we actually do this?

Reply:
Nope. I shall create this as a new PR